docs/rest_api.rst

```diff
@@ -72,6 +72,8 @@ Errors
 ======
 
 For any non-successful http status code, e.g. :http:statuscode:`409` or :http:statuscode:`400` there will be an accompanying ``errors`` field in the response json which you can check for more information on what went wrong with your request.
+However, when Raiden fails to process the incoming request and raises an exception, the returned http status code will be :http.statuscode:`500`. The caveat of this is that the response body will be just a string message which says "Internal server error".
```

Comment:
Perhaps mention that in almost all situations if you get 500 Error during normal usage it's a bug.

Reply:
> Perhaps mention that in almost all situations if you get 500 Error during normal usage it's a bug.

Not true, the server could just be shutting down

Reply:
That's why I said almost all. He already mentions the server shutting down instance in there.